api/graphql/resolvers/album.go

```diff
@@ -20,7 +22,9 @@ func (r *queryResolver) MyAlbums(ctx context.Context, order *models.Ordering, pa
 	return actions.MyAlbums(r.DB(ctx), user, order, paginate, onlyRoot, showEmpty, onlyWithFavorites)
 }
 
-func (r *queryResolver) Album(ctx context.Context, id int, tokenCredentials *models.ShareTokenCredentials) (*models.Album, error) {
+func (r *queryResolver) Album(ctx context.Context, id int, tokenCredentials *models.ShareTokenCredentials) (*models.Album,
+	error) {
```

Comment:
This line breaks weirly. Could we keep this in one line?

Reply:
done